server_addon/maya/server/settings/publishers.py

```diff
@@ -309,6 +386,169 @@ class ExtractAlembicModel(BaseSettingsModel):
         description="List of attribute prefixes for attributes that will be "
                     "included in the alembic export.",
     )
+    dataFormat: str = SettingsField(
```

Comment:
I understand that these are somewhat Alphabetically sorted - but I do feel it's more understandable to take some liberty and sort them somewhat more by category:

![image](https://github.com/ynput/ayon-core/assets/2439881/148d55a8-8023-4a6b-b1c7-eef58b021e57)

E.g. I would:
- move the attribute related settings together: bake attributes, custom attributes and user attributes
- move python and mel script callbacks together

That would likely also make the list of checkboxes sit nicer together without text fields in-between making it a cleaner list and easier to read.

Any thoughts?

Reply:
And potentially doing the same in the Publisher UI:

![image](https://github.com/ynput/ayon-core/assets/2439881/95c1dd45-fd71-4d6d-b2cf-221aabed5950)

May be nicer having the attributes settings lower there?

Reply:
Sure, done.